docs/install/scala_setup.md

```diff
@@ -0,0 +1,87 @@
+# Setup the MXNet Package for Scala
+
+The following instructions are provided for macOS and Ubuntu. Windows is not yet available.
+
+**Note:** If you use IntelliJ or a similar IDE, you may want to follow the [MXNet-Scala on IntelliJ tutorial](../tutorials/scala/mxnet_scala_on_intellij.html) instead of these instructions.
+
+## Setup Instructions
+**Step 1.** Download the MXNet source.
+
+```bash
+git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
+cd mxnet
+```
+
+**Step 2.** Install dependencies:
+
+**macOS Steps**
+
+```bash
+brew update
+brew tap caskroom/versions
+brew cask install java8
+brew install maven
+brew install opencv@2
+```
+
+**Ubuntu Steps**
+
+```bash
+sudo ./ci/docker/install/ubuntu_core.sh
+sudo ./ci/docker/install/ubuntu_scala.sh
+```
+
+**Step 3.** Run the demo MXNet-Scala project.
```

Comment:
demo application is only to validate the maven installation.Please change to

** Step 3. ** Validate your build.

You can validate your build by running the [ImageClassifier](https://github.com/apache/incubator-mxnet/tree/master/scala-package/examples/src/main/scala/org/apache/mxnetexamples/infer/imageclassifier) example

Reply:
This isn't a build from source page.

And that example doesn't actually work out of the box. It requires a full MXNet install plus env vars settings, etc. which doesn't work right now, so we're not putting it in the docs. (until it works)
 
I thought you (or someone) was providing a demo application that could be called very simply just by having the deps installed and by having that little xml snippet.

Let's review the flow... I'm posting an animation so you can see the install path and better understand the purpose of each page. Then we talk about what goes where.